mos_tests/nova/test.sh

```diff
@@ -2,8 +2,11 @@
 CONTR_ID=$(fuel node | grep controller | head -1 | awk '{print$1}')
 scp -rp ./../../mos_tests/ node-$CONTR_ID:~/
 
-ssh node-$CONTR_ID "export PYTHONPATH=.:$PYTHONPATH && source ~/openrc && nosetests mos_tests/nova/nova_tests.py --with-xunit --xunit-file=nova_tests_report.xml"
-scp node-$CONTR_ID:~/nova_tests_report.xml ~/
+ssh node-$CONTR_ID "export PYTHONPATH=.:$PYTHONPATH && source ~/openrc && nosetests mos_tests/nova/windows_compatibility_tests.py --with-xunit --xunit-file=windows_compatibility_tests_report.xml"
+scp node-$CONTR_ID:~/windows_compatibility_tests_report.xml ~/
+
+# ssh node-$CONTR_ID "export PYTHONPATH=.:$PYTHONPATH && source ~/openrc && nosetests mos_tests/nova/nova_tests.py --with-xunit --xunit-file=nova_tests_report.xml"
```

Comment:
Since I would like to separate execution of nova tests from execution of tests on windows compatibility.